pkg/store/driver/main_test.go

```diff
@@ -91,7 +91,7 @@ func createTiKVStore(t *testing.T) (kv.Storage, *domain.Domain) {
 }
 
 func createUnistore(t *testing.T) (kv.Storage, *domain.Domain) {
-	client, pdClient, cluster, err := unistore.New("", nil, nil)
+	client, pdClient, cluster, err := unistore.NewClassic("", nil)
```

Comment:
the related UT, such as `TestFailBusyServerCop`, also need to run in nextgen, this change will force this test only run in classic ?